draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -236,21 +236,16 @@ additional information about the cause of a DNS error.
 
 4. These extended error codes do not suffer from the limitations
 discussed in bullets (1) and (2), but the user still does not know the
-exact reason nor he/she is aware of the exact entity blocking the
+exact reason nor is aware of the exact entity blocking the
 access to the domain. For example, a DNS server may block access to a
 domain based on the content category such as "Malware" to protect the
 endpoint from malicious software, "Phishing" to prevent the user from
-revealing sensitive information to the attacker, etc. A user needs to
+revealing sensitive information to the attacker, etc. A user may need to
 know the contact details of the IT/InfoSec team to raise a complaint.
 
-5. When a DNS resolver or forwarder forwards the received EDE option, the
-EXTRA-TEXT field only conveys the source of the error ({{Section 3 of
-!RFC8914}}) and does not provide additional textual information about
-the cause of the error.
-
```

Comment:
We don't have to delete the line, it can be fixed as follows:
When a DNS resolver or forwarder forwards the received EDE option, the
EXTRA-TEXT field can convey the source of the error ({{Section 3 of
!RFC8914}}) and does not provide additional textual information about
the cause of the error.

Reply:
The point of the reviewer is that the EDE is an information about the cause itself. Which I think is OK. The key contrib of the spec is to structure the info and provide more details (c, j, etc.). 

This bullet does not bring much, IMO